packages/x-charts/src/ChartsXAxis/ChartsXAxis.tsx

```diff
@@ -222,6 +286,7 @@ function ChartsXAxis(inProps: ChartsXAxisProps) {
       style: {
         ...theme.typography.caption,
         fontSize: 12,
+        lineHeight: 1.25,
```

Comment:
The default line spacing is 1.66, which seems to be too large. 

Before:

![image](https://github.com/user-attachments/assets/53c2fd8f-3ad6-4ec2-8fb5-b6b0f1c0bc76)


After:

![image](https://github.com/user-attachments/assets/ee2322a6-69ae-44d6-912f-2b7aca4f6e40)